conda_devenv/devenv.py

```diff
@@ -137,7 +137,14 @@ def merge_dependencies_version_specifications(yaml_dict, key_to_merge):
         return
 
     # regex based on https://conda.io/docs/building/pkg-name-conv.html#package-naming-conventions
```

Comment:
Comment about the additon of channel specification or update with a new reference to some conda docs.

Reply:
I'm moving the comment so that it is specific only to the package name